### ISSUE TITLE
Initial selected item is not highlighted

### ISSUE THREAD

**Issue by sevcsik** [PolymerElements/paper-tabs#31]
When I set the `selected` attribute the first time (e.g. hardcoding it in the markup, or binding it to a property which has value whe the element is attached), the selected tab is not highlited with the selection bar.

If I changed the `selected` attribute afterwards, it works fine.

(version 1.0.0)

**Comment by sevcsik** [comment]
Looks like the problem is that for the first `_onTabChanged` call, [here](https://github.com/PolymerElements/paper-tabs/blob/master/paper-tabs.html#L407), we get a 0x0 rectangle:

```js
      var r = this.$.tabsContent.getBoundingClientRect();
```

which results in a division by zero few lines later.

**Comment by seahrh** [comment]
Yes selection bar is missing (on initial selection), even when I manually set `paper-tab` `focused` property to true.

**Comment by LOLKFC** [comment]
same issue here

**Comment by sevcsik** [comment]
I fixed this in PR https://github.com/PolymerElements/paper-tabs/pull/32 but I'm yet to add tests. I suspect that the same issue is causeing what @seahrh reports.

@seahrh, can you check it with my branch: https://github.com/sevcsik/paper-tabs/tree/patch-1?

**Comment by ktiedt** [comment]
Can we see about getting this merged? It has been months and it has now been hit by several users.

**Comment by ktiedt** [comment]
Worth noting, I worked around this in my case by adding:

```
this.$.tabs._selectSelected();
```

to the last line of my tabs parent elements attached() callback. I realize this is calling private API but it was a temporary (hopefully) solution to the problem.

**Comment by ronnyroeller** [comment]
+1

**Comment by cdata** [comment]
Hi, this issue should be fixed. Please re-open if it is not!

**Comment by ronnyroeller** [comment]
Still doesn't work for me.

**Comment by davetii** [comment]
Still not working

**Comment by PascalLaier** [comment]
In 2.0 this is still not working!

**Comment by thepassle** [comment]
Still not working.

The only workaround that works for me is calling notifyResize() in a setTimeout of 1 ms.

**Comment by katejeffreys** [comment]
@bicknellr @frankiefu @notwaldorf @e111077 this seems like it should be re-opened. Could one of you pls take a look when possible, or let me know who to ping? Thanks!

**Comment by srijken** [comment]
Indeed still not working. The notifyResize() from a setTimeout isn't a 100% work-around either. I've seen it work, but after adding some more (other) components, it broke, and I can't get it back :(

**Comment by thepassle** [comment]
The notifyResize() with a setTimeout also didnt solve it for all browsers i later found out.

We've recently switched our project to a build with webpack, and mysteriously the issue isnt present anymore. Maybe that points someone in the right direction?
